lib/azure/armrest/network/network_security_group_service.rb

```diff
@@ -0,0 +1,95 @@
+module Azure
+  module Armrest
+    # Class for managing network security groups.
+    class NetworkSecurityGroupService < ArmrestService
```

Comment:
Well, I think we always match the service type in the others. I would like to leave it if that's ok.